_data/simple-icons.json

```diff
@@ -10779,8 +10779,8 @@
         },
         {
             "title": "ŠKODA",
-            "hex": "4BA82E",
-            "source": "https://en.wikipedia.org/wiki/File:Skoda_Auto_logo_(2011).svg"
+            "hex": "0E3A2F",
+            "source": "https://www.skoda-storyboard.com/en/media-room/"
```

Comment:
```suggestion
            "source": "https://www.skoda.co.uk"
```

Reply:
How about we use skoda.com instead 💀

Reply:
skoda.com isn't one of their websites - we could use skoda-auto.com, but then we'd need to re-compare against that source. I'm happy to leave it as the `.co.uk` for now.

Reply:
I was joking 😛